upload-blocks/FileItem/FileItem.js

```diff
@@ -4,6 +4,9 @@ import { uploadFile } from '@uploadcare/upload-client';
 import { UiMessage } from '../MessageBox/MessageBox.js';
 import { fileCssBg } from '../svg-backgrounds/svg-backgrounds.js';
 import { customUserAgent } from '../utils/userAgent.js';
+import { ProgressBar } from './ProgressBar.js';
+
+ProgressBar.reg('file-item-progress-bar');
```

Comment:
Don't you think that we should register blocks in common place? We broke the uniform tag name generation in this case...